src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -46,13 +48,39 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+@dataclass
+class NavigatorVolumeMount:
+    """Describes EE volume mounts."""
+
+    src: str
+    dest: str
+    label: Optional[str] = None
+
+    def exists(self) -> bool:
+        """Does the volmount source exist?"""
+        return Path(self.src).exists()
+
+    def to_string(self) -> str:
+        """Renders the volume mount in a way that (docker|podman) understands."""
+        out = f"{self.src}:{self.dest}"
+        if self.label is not None:
+            out += f":{self.label}"
+        return out
```

Comment:
I think if we improve the var naming like this, it could add more clarity to the code:
```suggestion
        volume_mapping = f"{self.src}:{self.dest}"

        if self.label is None:
            return volume_mapping

        return f"{volume_mapping}:{self.label}"
```

Reply:
(comment for comment sake) I prefer the original because the label gets appended to the end of the vol mount, and I think the first shows that in a more straight forward fashion

Reply:
I don't see the label being appended, I see a contextless string being appended to some var called "out" that also does not have a designated semantical meaning.

Reply:
Would something like this be better, just so I grok it all.....

```
@dataclass
class NavigatorVolumeMount:
    """Describes EE volume mounts."""

    src: str
    #: The local source directory
    dest: str
    #: The mount point within the EE
    label: Optional[str] = None
    #: The label option(s) for the mount

    def exists(self) -> bool:
        """Does the volmount source exist?

        :return: A boolean indicating wether or not the src exists.
        """
        return Path(self.src).exists()

    def to_string(self) -> str:
        """Render the volume mount in a way that (docker|podman) understands.

        The resulting string consists of three fields, separated by colon characters (:).
        The fields must be in the correct order, and the meaning of each field is not immediately 
        obvious. In the case of bind mounts, 
        The first field is the path to the file or directory on the host machine.
        The second field is the path where the file or directory is mounted in the container.
        The third field is optional, and is a comma-separated list of options, such as ro, z, and Z.

        :return: A string formatted as a ``podman`` or ``docker`` volume mount string.
        """
        volume_mount_string = f"{self.src}:{self.dest}"
        if self.label is not None:
            volume_mount_string += f":{self.label}"
        return volume_mount_string
```

Reply:
(description based on container engine documentation)